howdoi/howdoi.py

```diff
@@ -41,17 +43,19 @@
 from howdoi import __version__
 from howdoi.errors import GoogleValidationError, BingValidationError, DDGValidationError
 
+logging.basicConfig(format='%(levelname)s: %(message)s')
+
 
-# rudimentary standardized 3-level log output
-def _print_err(err):
-    print(f'[ERROR] {err}')
+def error_logging(err):  # log level=error
```

Comment:
I think we can remove these three functions in favor of a simple call to `logging.error` or whatever the underlying function will be.

Reply:
Done